packages/app/playwright.config.ts

```diff
@@ -31,7 +37,22 @@ export const playwrightConfig: PlaywrightTestConfig = {
     screenshot: 'only-on-failure',
     headless: false,
   },
-  // ignore lock test because it takes too long and it will be tested in a separate config
+  projects: [
+    {
+      name: 'chromium',
+      use: {
+        ...devices['Desktop Chromium'],
+      },
+    },
+    {
+      name: 'chrome-beta',
+      use: {
+        channel: 'chrome-beta',
+        ...devices['Desktop Chrome'],
+      },
+      retries: IS_CI ? 4 : 0,
```

Comment:
CI shouldn't retry more than once to avoid long wait times. Check if we still need this with the latest fixes